website/content/en/docs/deployment/cognito/guide-terraform.md

```diff
@@ -75,27 +75,23 @@ pwd
     EOF
     ```
 
-### All Configurations
-
-A full list of inputs for the terraform stack can be found [here](https://github.com/awslabs/kubeflow-manifests/blob/main/deployments/cognito/terraform/variables.tf).
+### List all configurations
```

Comment:
I see how this is confusing, we want to tell the user, there are more configurations and they can see those in this file. What do you think is an appropriate title for that? `List all configurations` and `all Configurations` both are equally confusing

maybe View all configurations?